src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -50,6 +50,20 @@ class ConfigurationHelper implements Serializable {
         return mixin(stepConfiguration, filter, compatibleParameters)
     }
 
+    ConfigurationHelper convertToAbsolutePaths(Set pathConfigurationKeys){
+        if(pathConfigurationKeys){
+            //Copy configuration to not persist any changes done below
+            this.config = MapUtils.deepCopy(this.config)
```

Comment:
the copy overrides the original value? I don’t get what this does and why it is required.

Reply:
The original value could still contain a reference to the data source, e.g. default configuration. If we now change a path to the absolute value this change could also be propagated to the data source, e.g. default configuration.